src/aria/widgets/form/list/ListController.js

```diff
@@ -499,8 +499,10 @@ module.exports = Aria.classDefinition({
         calcMoveFocus : function (flowOrientation, focusIndex, numberOfRows, numberOfColumns, keyCode, numberItems) {
 
             var moveFocus = focusIndex;
+            var cfg = this._data.cfg;
+            var waiAria = cfg && cfg.waiAria;
 
-            if (numberOfColumns == 1) {
+            if (numberOfColumns == 1 || waiAria) {
```

Comment:
Is it really needed to add this line?
Because `numberOfColumns` is already set to `1` in the `$constructor` of the `List` class.